### ISSUE TITLE
[Request] Add suggestions to Extensions for existing features

### ISSUE THREAD

**Issue by bernaferrari** [microsoft/vscode#87942]
Today I found out Visual Studio Code has native auto save.
I tried searching in `Extensions` for extensions that did this, but couldn't find any. Actually, for the keyword "autosave" there is only one result and it is auto-save for window changes. I ended up going to Google and figuring out it was a simple File -> Auto Save toggle.

<img src="https://user-images.githubusercontent.com/351125/71631721-fc171680-2be8-11ea-8771-36ce6e760f12.png" height="200px">

So, I was thinking... what if VSCode had auto suggestions in the extension section for things that it already has? It could show up an item saying "looking for auto-save? Here is how to enable it". "looking for shortcuts? Go here to edit them", "themes? This is the place" and so on.. It would get way more accessible and intuitive than currently is.

**Comment by egamma** [comment]
@bernaferrari making VS Code features easily discoverable is an important goal. Auto save is one of these features and we have invested to make it more discoverable. Obviously you haven't discovered the menu item `File>Auto Save` and you haven't found the corresponding setting by doing a settings search:

![image](https://user-images.githubusercontent.com/172399/71644201-cd6e6e00-2cc4-11ea-9129-f9e8816c44fe.png)

I assume you though that `auto save` is contributed by an extension as it is done in other editors?

**Comment by bernaferrari** [comment]
I actually went straight into Extensions, didn't even look at Settings (I really really dislike VSCode Settings window and feel it is not reliable or accessible at all. But that is a talk for another day/issue or feel free to ping me elsewhere).

With Atom, there is [an extension](https://atom.io/packages/autosave) that does it (342k downloads). With Sublime, there is [also an extension](https://packagecontrol.io/packages/auto-save) for it (113k downloads). That's why I thought VSCode also had an extension, but it didn't have, so I needed to Google for my answer. Which why I think adding some discoverability features in extensions would help.

**Comment by egamma** [comment]
>With Atom, there is an extension that does it (342k downloads). With Sublime, there is also an extension for it (113k downloads). That's why I thought VSCode also had an extension,

Makes sense, one way to address is to have some proxy extensions in the marketplace for features that is built-into VS Code but not for other editors: auto save, emmet, terminal. This needs more thoughts.
